cmd/packaging/darwin-dmg.go

```diff
@@ -31,25 +31,30 @@ var DarwinDmgTask = &packagingTask{
 			return "", err
 		}
 
-		var cmdGenisoimage *exec.Cmd
+		var cmdCreateBundle *exec.Cmd
 		switch os := runtime.GOOS; os {
 		case "darwin":
-			cmdGenisoimage = exec.Command("hdiutil", "create", "-volname", packageName, "-srcfolder", "dmgdir", "-ov", "-format", "UDBZ", outputFileName)
+			cmdCreateBundle = exec.Command("hdiutil", "create", "-volname", packageName, "-srcfolder", "dmgdir", "-ov", "-format", "UDBZ", outputFileName)
 		case "linux":
-			cmdGenisoimage = exec.Command("genisoimage", "-V", packageName, "-D", "-R", "-apple", "-no-pad", "-o", outputFileName, "dmgdir")
+			cmdCreateBundle = exec.Command("mkisofs", "-V", packageName, "-D", "-R", "-apple", "-no-pad", "-o", outputFileName, "dmgdir")
```

Comment:
This is not in the PR description :grin: 
Why move to mkisofs? (I don't know anything about genisoimage vs mkisofs, just curious here)

Reply:
`genisoimage` is a fork of `mkisofs` which is really only used in Ubuntu. To make it compatible for more people I changed it. I also now noticed that I forgot to change to docker image to make it working. But you are right it probably shouldn't be changed in this PR.

Reply:
I added the symlink to the Dockerfile. I would still change from `genisoimage` to `mkisofs` in this PR if that is ok for you

Reply:
Instead of asking users to create a symlink, we could first look for the alternative (genisoimage) and use that if it is available. But that's something for a different PR